.github/workflows/release.yml

```diff
@@ -10,31 +10,45 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    # We don't attempt to cross-compile rust binaries from one OS to another.
+    # So just spin up a runner on each OS to build its binaries.
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Mount bazel caches
-        uses: actions/cache@v3
```

Comment:
note, this cache setup is old now, we have something coming to bazel-contrib/setup-bazel.

For now a clean build isn't all that slow.